### ISSUE TITLE
Offline mode lacks player skins

### ISSUE THREAD

**Issue by Omgise** [Roadhog360/SimpleSkinBackport#8]
![屏幕截图 2024-09-21 140025](https://github.com/user-attachments/assets/cfddd881-b2a0-48f5-ba1c-1e605fb2fba0)
![屏幕截图 2024-09-21 140035](https://github.com/user-attachments/assets/67aafcea-4027-4e61-8cf9-56a672a73af1)

**Comment by Roadhog360** [comment]
What other mods do you have installed that may be touching skins? Be sure you don't have another skin backport installed.

**Comment by Omgise** [comment]
Just using SimpleSkinBackport is still the same
![image](https://github.com/user-attachments/assets/bb8ca58c-d96d-40a2-91ec-f1db595e4105)
![image](https://github.com/user-attachments/assets/b7001a6a-4e66-48a2-813c-fb90990bfebe)
![image](https://github.com/user-attachments/assets/0c932ec0-ee34-4e47-86d5-f03aac5332c3)

**Comment by Roadhog360** [comment]
![image](https://github.com/user-attachments/assets/47b5761d-d653-462b-adc2-31f8e218a100)
Note that skin serving only works on online mode; this is a vanilla issue and is not related to SimpleSkinBackport. If you are running offline mode I will close this issue.

**Comment by Roadhog360** [comment]
As a side note, please let me know if this fix mod works for you! https://www.curseforge.com/minecraft/mc-mods/lan-host-skin-re-fix-ed/files/2890211

I may roll this fix into SimpleSkinBackport in the future if I get permission to.

**Comment by Roadhog360** [comment]
I should note I am aware the LAN host skin fix isn't exactly the same issue, but I feel like it MIGHT work since it fixes a similar issue that causes skins not to load, might fix your issue too.

**Comment by Omgise** [comment]
Yes, it works, players in online mode can now display their skins normally :D
![image](https://github.com/user-attachments/assets/1aee8fcf-fa90-40c6-a4ae-db5265e1367e)
![image](https://github.com/user-attachments/assets/fedcba32-0af3-4f9e-8054-0a917c7a6753)

**Comment by Roadhog360** [comment]
It's a bit weird how the different default skin is not shown on the other account. I'll look into this. Despite me saying this is a vanilla issue, I've decided to leave it open as I'm interested in fixing it now.
